lucetc/src/decls.rs

```diff
@@ -347,12 +368,12 @@ impl<'a> ModuleDecls<'a> {
         }
     }
 
-    pub fn get_module_data(&self) -> ModuleData {
+    pub fn get_module_data(&self) -> Result<ModuleData, LucetcError> {
```

Comment:
Just curious why this was changed to be fallible. The current method body doesn't ever throw an error.

Reply:
ugh. shouldn't be, here. I had some conflict resolution issues with another branch (adding types to module data, where it's technically possible that cranelift declares a signature to us that isn't representable with wasm types)